R/deployApp.R

```diff
@@ -211,10 +211,14 @@ deployApp <- function(appDir = getwd(),
   taskId <- if (is.null(task$task_id)) task$id else task$task_id
   response <- client$waitForTask(taskId, quiet)
   if (!is.null(response$code) && response$code != 0) {
+    # wait 1/10th of a second for any queued output get picked by RStudio
+    # before emitting this, to ensure it's the last line the user sees
+    Sys.sleep(0.10)
```

Comment:
nit: move this before the code check and you only need one sleep statement. otherwise :+1: